portfolio/src/main/java/com/google/sps/servlets/VideoServlet.java

```diff
@@ -36,4 +35,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String paramString = "?q=" + query + "&score=" + sentimentScore + "&title=" + title;
         response.sendRedirect("results.html" + paramString);
     }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
```

Comment:
I recently moved my code to doPost since it's going to be storing analysis data soon. Just keep that in mind when you merge.